.goreleaser.yml

```diff
@@ -9,13 +9,16 @@ builds:
       - linux
       - windows
       - darwin
-# List of combinations of GOOS + GOARCH + GOARM to ignore.
+    # List of combinations of GOOS + GOARCH + GOARM to ignore.
     # Default is empty.
     ignore:
       - goos: darwin
         goarch: 386
       - goos: darwin
```

Comment:
Unrelated to this CL. But looks like we ignore Mac on ARM64? @naveensrinivasan FYI